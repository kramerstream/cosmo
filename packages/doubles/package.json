{
  "name": "@kramertest/cosmo-doubles",
  "version": "0.1.0",
  "description": "Test doubles library.",
  "homepage": "http://kramer.stream",
  "author": {
    "name": "KramerStream",
    "email": "hello@kramer.stream",
    "url": "http://kramer.stream"
  },
  "contributors": [
    {
      "name": "Raúl G. González",
      "email": "raulggonzalez@kramer.stream",
      "url": ""
    }
  ],
  "keywords": [
    "assert",
    "test"
  ],
  "bugs": {
    "url": "https://github.com/kramerstream/assert/issues",
    "email": "doubles@kramer.stream"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/kramerstream/assert.git"
  },
  "preferGlobal": false,
  "main": "lib/index.js",
  "files": [
    "lib/"
  ],
  "engines": {
    "node": ">= 10.19",
    "npm": ">= 6.14"
  },
  "dependencies": {
    "@dogmalang/core": "^1.0.0-rc12.0",
    "lodash.isequal": "^4.5.0"
  },
  "devDependencies": {
    "@babel/cli": "^7.12",
    "@babel/core": "^7.12",
    "@babel/preset-env": "^7.12",
    "@ethronpi/babel": "^0.11.0",
    "@ethronpi/eslint": "^0.1.0",
    "@ethronpi/exec": "^0.2.0",
    "@ethronpi/fs": "^0.3.0",
    "@ethronpi/npm": "^0.3.0",
    "@kramertest/cosmo-expected": "^0.1.0",
    "eslint": "^7.14.0",
    "nyc": "^15.1.0"
  },
  "scripts": {
    "build": "ethron r build",
    "test": "ethron r test",
    "cov": "ethron r build && node_modules/.bin/nyc ethron r test",
    "pipelines": "node_modules/.bin/nyc ethron r test"
  },
  "nyc": {
    "check-coverage": true,
    "cache": true,
    "lines": 100,
    "branches": 100,
    "statements": 100,
    "functions": 100,
    "exclude": [
      "**/test/**/*.js",
      "**/__test__/**/*.js"
    ],
    "reporter": [
      "lcov",
      "text"
    ]
  }
}
