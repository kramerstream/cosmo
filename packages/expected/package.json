{
  "name": "@kramertest/cosmo-expected",
  "version": "0.1.0",
  "description": "Assertion test library.",
  "homepage": "http://kramer.stream",
  "author": {
    "name": "KramerStream",
    "email": "hello@kramer.stream",
    "url": "http://kramer.stream"
  },
  "contributors": [
    {
      "name": "Raúl G. González",
      "email": "raulggonzalez@kramer.stream",
      "url": ""
    }
  ],
  "keywords": [
    "assert",
    "test"
  ],
  "bugs": {
    "url": "https://github.com/kramerstream/assert/issues",
    "email": "assert@kramer.stream"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/kramerstream/assert.git"
  },
  "preferGlobal": false,
  "main": "lib/index.js",
  "files": [
    "lib/"
  ],
  "engines": {
    "node": ">= 10.19",
    "npm": ">= 6.14"
  },
  "dependencies": {
    "@dogmalang/core": "^1.0.0-rc12.0"
  },
  "devDependencies": {
    "@babel/cli": "^7.12",
    "@babel/core": "^7.12",
    "@babel/preset-env": "^7.12",
    "@dogmalang/path": "^0.1.0",
    "@ethronjs/assert": "^0.1.2",
    "@ethronpi/babel": "^0.11.0",
    "@ethronpi/eslint": "^0.1.0",
    "@ethronpi/exec": "^0.2.0",
    "@ethronpi/fs": "^0.3.0",
    "@ethronpi/npm": "^0.3.0",
    "chai": "^4.2.0",
    "eslint": "^7.14.0",
    "nyc": "^15.1.0"
  },
  "scripts": {
    "build": "ethron r build",
    "test": "ethron r test",
    "cov": "ethron r build && node_modules/.bin/nyc ethron r test",
    "pipelines": "node_modules/.bin/nyc ethron r test"
  },
  "nyc": {
    "check-coverage": true,
    "cache": true,
    "lines": 100,
    "branches": 100,
    "statements": 100,
    "functions": 100,
    "exclude": [
      "**/test/**/*.js",
      "**/__test__/**/*.js"
    ],
    "reporter": [
      "lcov",
      "text"
    ]
  }
}
